tests/foreman/api/test_environment.py

```diff
@@ -145,38 +131,33 @@ def test_positive_update_name(self):
                 self.assertEqual(env.name, new_name)
 
     @tier2
-    def test_positive_update_org(self):
+    def test_positive_update_and_remove(self):
         """Update environment and assign it to a new organization
+        and location. Delete env
 
         :id: 31e43faa-65ee-4757-ac3d-3825eba37ae5
 
-        :expectedresults: Environment entity is updated properly
+        :expectedresults: Environment entity is updated and removed
+            properly
+
+        :CaseImportance: Critical
 
         :CaseLevel: Integration
         """
         env = entities.Environment().create()
```

Comment:
Can you please add an assertion here environment is not having organisation and location by default. Like many cases, we add some properties from nailgun by default.

This check will make sure that Environment is not created having org or location.  
e.g. 

```
self.assertEqual(len(env.organization), 0)
self.assertEqual(len(env.location), 0)
```